Jan 6 11:54:21 AM  (node:53) UnhandledPromiseRejectionWarning: Error: missing revert data in call exception; Transaction reverted without a reason string [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (data="0x", transaction={"to":"0x762C566e21B65E9377CC4FC45D91A24530308bd8","data":"0x4698156a","accessList":null}, error={"reason":"bad response","code":"SERVER_ERROR","status":503,"headers":{"date":"Fri, 06 Jan 2023 10:54:21 GMT","content-type":"text/html","content-length":"190","connection":"close","x-frame-options":"SAMEORIGIN","x-xss-protection":"1; mode=block","x-content-type-options":"nosniff","referrer-policy":"origin-when-cross-origin","strict-transport-security":"max-age=31536000; includeSubdomains"},"body":"<html>\r\n<head><title>503 Service Temporarily Unavailable</title></head>\r\n<body>\r\n<center><h1>503 Service Temporarily Unavailable</h1></center>\r\n<hr><center>nginx</center>\r\n</body>\r\n</html>\r\n","requestBody":"{\"method\":\"eth_call\",\"params\":[{\"to\":\"0x762c566e21b65e9377cc4fc45d91a24530308bd8\",\"data\":\"0x4698156a\"},\"latest\"],\"id\":88,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"https://data-seed-prebsc-1-s1.binance.org:8545/"}, code=CALL_EXCEPTION, version=providers/5.7.2)
Jan 6 11:54:21 AM      at Logger.makeError (/opt/render/project/src/node_modules/@ethersproject/logger/lib/index.js:238:21)
Jan 6 11:54:21 AM      at Logger.throwError (/opt/render/project/src/node_modules/@ethersproject/logger/lib/index.js:247:20)
Jan 6 11:54:21 AM      at checkError (/opt/render/project/src/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:108:16)
Jan 6 11:54:21 AM      at JsonRpcProvider.<anonymous> (/opt/render/project/src/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:751:47)
Jan 6 11:54:21 AM      at step (/opt/render/project/src/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:48:23)
Jan 6 11:54:21 AM      at Object.throw (/opt/render/project/src/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:29:53)
Jan 6 11:54:21 AM      at rejected (/opt/render/project/src/node_modules/@ethersproject/providers/lib/json-rpc-provider.js:21:65)
Jan 6 11:54:21 AM      at processTicksAndRejections (internal/process/task_queues.js:95:5)
Jan 6 11:54:21 AM  (Use `node --trace-warnings ...` to show where the warning was created)
Jan 6 11:54:21 AM  (node:53) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
Jan 6 11:54:21 AM  (node:53) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.